.buildkite/shared-pipeline-vars

```diff
@@ -5,8 +5,8 @@
 
 # The ~> modifier is not currently used, but we check for it just in case
 XCODE_VERSION=$(sed -E -n 's/^(~> )?(.*)/xcode-\2/p' .xcode-version)
-CI_TOOLKIT_VERSION='3.10.0'
-NVM_PLUGIN_VERSION='0.3.0'
+CI_TOOLKIT_VERSION='mokagio/fail-early-on-aws-missing-credentials'
```

Comment:
Would it be safer to wait for the CI Toolkit release?

Reply:
Yes. It would be best. In other PRs, e.g. https://github.com/Automattic/pocket-casts-android/pull/3653/files, I left a note about it, but I forgot here.